### ISSUE TITLE
REST endpoint /V1/store/storeViews is missing is_active value in store data

### ISSUE THREAD

**Issue by hatimeria-artur-jewula** [magento/magento2#10922]
When doing API call to /rest/default/V1/store/storeViews the result list items do not show is_active value

### Preconditions
1. Any Magento2 version

### Steps to reproduce
1. Perform a rest call to http://store.domain/rest/[store-view-code]/V1/store/storeViews

### Expected result

1. The resulted data should include is_active value for the store, ie.
```
{
    "id": 1,
    "code": "default",
    "name": "English",
    "website_id": 1,
    "store_group_id": 1,
    "is_active": 1,
    "extension_attributes": {...}
},...
```

### Actual result
1. Result is missing is_active flag, ie.
```
{
    "id": 1,
    "code": "default",
    "name": "English",
    "website_id": 1,
    "store_group_id": 1,
    "extension_attributes": {...}
},...
```

**Comment by magento-engcom-team** [comment]
@hatimeria-artur-jewula thank you for your bug report.
We've created internal ticket MAGETWO-75448 to track progress on the issue.

**Comment by orlangur** [comment]
`\Magento\Store\Api\StoreRepositoryInterface::getList` is not designed to contain such information.

`\Magento\Store\Api\StoreRepositoryInterface::getActiveStoreById` + `\Magento\Store\Api\StoreRepositoryInterface::getById` looks pretty sufficient to me to cover the `is_active` aspect.

On the other hand, if your application needs `is_active` in listing for some scenario, it can be added easily as extension attribute.

@magento-engcom-team @ishakhsuvarov thoughts?

**Comment by hervebackstage** [comment]
I can work on this issue?

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-75322

**Comment by sdzhepa** [comment]
Hello @hatimeria-artur-jewula 

I found the issue was fixed 

Should be available for 2.3.0, see commits for more details:
- f164cbc03f43e3ec519b9006ca9eefb14df5831f

This ticket can be closed.
Please, feel free to reopen or create a new one if the issue still exists or was not fully fixed

Thank you for feedback and collaboration
